.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -50,6 +56,14 @@ If no open issue exists, you can open one here:
 https://github.com/uswds/uswds/issues/new/choose.
 -->
 
+## Related pull requests
+
+_Indicate if there are other pull requests necessary to complete this issue_
+
+<!--
+Some changes to the USWDS codebase require a change to the documentation site, and need a pull request in the uswds-site repo. This could include new or updated component documentation, new or updated settings documentation, or changelog entries. Add links to any related PRs in this section. If this change requires an update to the uswds-site repo, but that PR does yet exist, just make sure to note that here.
+-->
+
```

Comment:
...and I added a new section to either link a related PR (or PRs) for documentation, etc, or at the very least, indicate that it's necessary but not yet done